index.js

```diff
@@ -7,41 +7,56 @@ var express = require('express');
 var bodyParser = require('body-parser');
 var stream = require('stream');
 var figlet = require('figlet');
+var basicAuth = require('basic-auth-connect');
+var compress = require('compression');
 
 var yargs = require('yargs')
     .usage('usage: $0 [options] <aws-es-cluster-endpoint>')
     .option('b', {
         alias: 'bind-address',
-        default: '127.0.0.1',
+        default: process.env.BIND_ADDRESS || '0.0.0.0',
```

Comment:
You can change it, its configurable. :)
When I said its not telling that is binding to 0.0.0.0 is bc there's anywhere with "console.log" or anything else printing "I'm going to bind it to 0.0.0.0"